cmd/pbstatuser/main.go

```diff
@@ -162,9 +197,13 @@ func checkSourceAvailabilityGCP(ctx context.Context) {
 			sr.Err = err
 			logger.Warn().Msgf("Could not list gcp regions %s", err)
 			chSend <- sr
+			totalAvailabilityCheckReqs.WithLabelValues("gcp", string(sr.Status), err.Error()).Inc()
+			availabilityCheckReqsDuration.WithLabelValues("gcp", err.Error()).Observe(float64(time.Since(start).Nanoseconds()) / 1000000)
 		} else {
 			sr.Status = kafka.StatusAvaliable
 			chSend <- sr
+			totalAvailabilityCheckReqs.WithLabelValues("gcp", string(sr.Status), "").Inc()
+			availabilityCheckReqsDuration.WithLabelValues("gcp", "").Observe(float64(time.Since(start).Nanoseconds()) / 1000000)
```

Comment:
Okay this is getting out of control, we see this bit so many times that I believe this definitely needs a helper function that will return you this. I think the best place for such function would be the newly created "metrics" package: `metrics.TimeSinceMs(start)` (returns float).